src/EditorFeatures/Core/Remote/SolutionChecksumUpdater.cs

```diff
@@ -21,115 +19,148 @@ namespace Microsoft.CodeAnalysis.Remote
     /// This class runs against the in-process workspace, and when it sees changes proactively pushes them to
     /// the out-of-process workspace through the <see cref="IRemoteAssetSynchronizationService"/>.
     /// </summary>
-    internal sealed class SolutionChecksumUpdater : GlobalOperationAwareIdleProcessor
+    internal sealed class SolutionChecksumUpdater
     {
         private readonly Workspace _workspace;
-        private readonly TaskQueue _textChangeQueue;
-        private readonly AsyncQueue<IAsyncToken> _workQueue = new();
```

Comment:
prior system used queues without any ability for batching/debouncing.  New system can do that.